src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+linkTitle: Creating a Grafana organization
+title: Creating a Grafana organization
+description: Guide explaining how to manage Grafana organizations in the Observability Platform.
+menu:
+  principal:
+    identifier: tutorials-observability-multitenancy-create-grafana-organization
+    parent: tutorials-observability-multitenancy
+weight: 40
+last_review_date: 2024-12-12
+user_questions:
+  - How to create a grafana organization?
+  - How to access multi-tenant observability data?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+When you first access [your installations Grafana]({{< relref "/tutorials/observability/data-exploration/accessing-grafana" >}}), you will be accessing an organization named _Shared Org_ which gives you access to a set of curated dashboards. However, if you want to be able to provide multi-tenancy of your observability data (data isolation between tenants), the Observability Platform allows you to create your own organizations in self-service.
+
+## Creating your own organization
+
+To create your own Grafana organization, you can create a [`GrafanaOrganization`](https://raw.githubusercontent.com/giantswarm/observability-operator/refs/heads/main/config/crd/observability.giantswarm.io_grafanaorganizations.yaml) custom resource in the management cluster containing the Grafana organization specification.
```

Comment:
```suggestion
To create your own `Grafana` organization, you can create a [`GrafanaOrganization`](https://raw.githubusercontent.com/giantswarm/observability-operator/refs/heads/main/config/crd/observability.giantswarm.io_grafanaorganizations.yaml) custom resource in the management cluster containing the `Grafana` organization specification.
```

Reply:
I don't think Grafana should be in a code block here

Reply:
yeah this is a bit controversial. Or we use lowercase without quotes or we use quotes and uppercase. (Following MS style guide). Now, we have used quotes for project names. I am not proud but I want to unify it, we can change it though but not in this PR please

Reply:
Grafana is not a technical concept here, it's a trademarked name of a product. Just as Kubernetes is [not code-blocked in Kubernetes documentation](https://kubernetes.io/docs/contribute/style/style-guide/) (and shouldn't be in ours), Grafana needs no special handling here IMO, also [according to MS](https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-common-text-elements).

Reply:
hmm, what I meant is our guidelines encourage [lower case nouns](https://intranet.giantswarm.io/docs/content/style-guide/#capitalization) so I tend to use lowercase. In case of project/tools I have used title plus backtick. In MS guide style opt for title case but they dont say anything about quoting. The decision of quoting is highlight the tool (we did one introducing it firs time and I found contradictory not doing it always. But I may be too strict on that, I happy to remove backticks in followup PR (you are not the first asking for it)